### ISSUE TITLE
Summary ideas Metabolomics 

### ISSUE THREAD

**Issue by oliveralka** [OpenMS/OpenMS#5336]
@FabianAicheler added some ideas recently which we will some up here, as an issue until we decide if a project should be initiated. 

- More interactivity/dynamic result processing in KNIME? E.g., the MS view Javascript prototype, other interactive Javascript nodes, combined with interactive tables, how close can we get during the evaluation/filtering/discovery/annotation phase (i.e., once working on tables) to Commercial solutions. For example: Given featureXmls, consensuXmls, idXmls, AMS/Sirius results can we integrate this to allow dynamic filtering/ranking/annotation of putative metabolite candidates?

- Improve how to integrate Experimental Design/Sirius/MAD knowledge with each other (and AMS,MapAligners,FL). To put everything together into a nicer metabolomics tutorial workflow (Also look at Optimus for inspiration concerning workflow variables, automation, control flow?). I.e., more on how to nicely integrate the different outputs and reduce complexity visible to standard users?

- Do some benchmarking for such an integrated approach? Collect benchmark data? Datasets I'm aware of (Fabian): CAMERA dataset (only MS1, plant background with spike-ins, at some time I spent/wasted time on manually annotating the features corresponding to the standards; Oliver Alka has some pesticide data he evaluated in-depth (also MS2, around 250 pesticides in the project db); https://doi.org/10.1016/j.aca.2018.05.001 Recent metabolomics benchmarking publication "standard mixtures benchmark dataset consisting of 1100 compounds with specified concentration ratios including 130 compounds with significant variation of concentrations"

**Comment by jpfeuffer** [comment]
I am happy to work on that with you guys.

**Comment by jpfeuffer** [comment]
Can someone convert the Sciex data to mzML? We do not have a Windows machine with enough space (70GB plus result of conversion). I do not want to risk our Jenkins builder, which would be close to full then.

**Comment by jpfeuffer** [comment]
Wait, the wiff files are 4MB??

**Comment by cbielow** [comment]
@jpfeuffer try PCPool machines or "copy"/"paste" Terminal server? You can mount any network drive there...

**Comment by jpfeuffer** [comment]
True, I will try that, but I think there is something wrong with the [upload](https://www.ebi.ac.uk/metabolights/MTBLS736/files):
The wiff files are only 4MB and MSconvert gives me:

```
Failed - System.Exception: [ExperimentImpl::initializeBPC()] Error processing SampleA_1.wiff (sample 1).
Could not get spectrum for C:\Users\builder\Downloads\SampleA_1.wiff (sample 1)
(Period = 1, Experiment = 1, Cycle = 1).

Could not open data stream. Is a required 'scan' file missing?
```
